docs/messaging-service/sms-config.md

```diff
@@ -0,0 +1,28 @@
+# SMS gateway configuration
+
+Messaging service supports 2 ways of sending SMS messages out of the box:
+
+- By integrating with the [CM.com SMS gateway](https://www.cm.com/sms/)
+- By integrating with a custom API
+
+## CM.com integration configuration
+
+The default integration for the Messaging service is with CM.com. CM.com requires
+a [product token](https://www.cm.com/app/docs/en/api/business-messaging-api/1.0/index#integration-authentication) to send SMS messages via its gateway.
+Please [contact support](https://onewelcome.atlassian.net/servicedesk/customer/portals) to obtain a product token from CM.com if you do not have one. To use it,
```

Comment:
... to obtain a product token from CM.com, if you do not have one already. To be able to use it,
the product token must be specified in the configuration screen.

Reply:
fixed